pkg/cmd/admin/router/router.go

```diff
@@ -56,8 +56,10 @@ you have failover protection.`
   $ %[1]s %[2]s router-west --credentials=/path/to/openshift-router.kubeconfig --service-account=myserviceaccount --replicas=2
 
   # Use a different router image and see the router configuration
-  $ %[1]s %[2]s region-west -o yaml --credentials=/path/to/openshift-router.kubeconfig --service-account=myserviceaccount --images=myrepo/somerouter:mytag`
+  $ %[1]s %[2]s region-west -o yaml --credentials=/path/to/openshift-router.kubeconfig --service-account=myserviceaccount --images=myrepo/somerouter:mytag
 
+  # Run the router with a hint to the underlying implementation to _not_ expose statistics.
+  $ %[1]s %[2]s router-west --credentials=/path/to/openshift-router.kubeconfig --service-account=myserviceaccount --stats-port=0`
```

Comment:
is the help text string terminator - it ends the help text for  ```routerLong```.
Will move to the next line for clarity.   

And will add a check in the e2e test to verify the port is open.